sherpa/fit.py

```diff
@@ -1118,7 +1117,7 @@ def calc_stat(self):
 
         See Also
         --------
-        calc_chisqr, calc_stat_info
+        `calc_chisqr`, `calc_stat_info`
```

Comment:
Do we really have to add these now? You can see that these used to generate links, without the backticks - e.g. https://sherpa.readthedocs.io/en/latest/fit/api/sherpa.fit.Fit.html#sherpa.fit.Fit.calc_stat and I'm just hoping we can save some time

Reply:
No, we don't. Thanks for pointing that out. In fact, we can remove even more of the backticks in other locations.